eventlet/wsgi.py

```diff
@@ -510,7 +513,7 @@ def write(data):
                     self.close_connection = 1
 
                 if 'content-length' not in header_list:
-                    if self.request_version == 'HTTP/1.1':
+                    if not bodyless[0] and self.request_version == 'HTTP/1.1':
```

Comment:
I think this will interfere with pipelining requests/responses. We may want something more like
```
if bodyless[0]:
    pass  # client doesn't expect a body anyway
elif self.request_version == 'HTTP/1.1':
    use_chunked[0] = True
    towrite.append(b'Transfer-Encoding: chunked\r\n')
elif 'content-length' not in header_list:
    self.close_connection = 1
```
(Side note: can that last block become an `else:`? Didn't we just check that `'content-length'` is not in `header_list`?)